Changes phoron gas to purple (Alongside various related changes of grape variety.)

chaoko99 opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->
-Nameswapped the purple and orange phoron sprites.
-The phoron tank, phoron canister, phoron pressure tank (Both used and unused varieties), the tank ont he flamethrower, the TTV phoron tank, the tank in the radiation collector -- Are all grape flavored (Sprite recolors.)
![ss 2017-03-17 at 05 13 52](https://cloud.githubusercontent.com/assets/4706052/24067337/60be547e-0b38-11e7-954d-af9600d90029.png)

chaoko99 committed:
Makes the canister.dm reference a slightly more accurate sprite

chaoko99 committed:
Purple pressuretanks.

chaoko99 committed:
Purple flamethrower tanks!

chaoko99 committed:
Purple phoron tank sprite

chaoko99 committed:
Grape ape radiation collector.

chaoko99 committed:
Corrections.

chaoko99 committed:
Purple TTV tank.

chaoko99 committed:
Grapel popped canister

chaoko99 committed:
grape flavored atmospherics.

chaoko99 committed:
Changelog.

MistakeNot4892 commented:
As far as I am aware the canisters are orange because of being explosively flammable, not because the contents are orange.

chaoko99 commented:
If I missed any of the obviously related sprites, do please let me know?

chaoko99 commented:
I initially tried to just swap out the overlay sprites, but I was told that I'd be better off either making everything phoron related orange, OR, to make the gaseous phoron related sprites purple. I chose the easier option.

Kasuobes commented:
Purple is just a deeper shade of blue, doesn't stand out compared to oxygen.

Just stick with the mental association of red/yellow/black = danger, blue/green/white = safe. Who cares if the can doesn't match its contents.

Kasuobes commented:
Or if you want to do it properly recolor everything according to this https://en.wikipedia.org/wiki/Bottled_gas#European_cylinder_colours

chaoko99 commented:
@Kasuobes 
![image](https://cloud.githubusercontent.com/assets/4706052/24067511/2a404170-0b3b-11e7-8f99-e79000a8b0e9.png)

Oxygen is marked differently, at any rate. However, I might just slap some hazard stripes on if I absolutely must.. Or perhaps even do the complete canister resprite you suggest.

chaoko99 committed:
Fixes changelog.

Okay how.

Kasuobes commented:
The color of the can/tank matters very little, it's the identification labels/markings that matter. For something as dangerous as phoron the little "P" doesn't really make it distinct, neither is the the closeness of purple to blue.

I'll support recoloring all the cans. N2 being red is just fundamentally dumb.

chaoko99 commented:
@Kasuobes I'd like some suggestions on how I'd go about that, from a design standpoint that is. Would it be best to put the stripes on the top of the tank, or..?

And here's an idea I was playing with.
![image](https://cloud.githubusercontent.com/assets/4706052/24067989/4b5e67d2-0b41-11e7-9be3-62a1cada0727.png)

Kasuobes commented:
Black red yellow, must be some quality German engineering.

Markings and decals should always be on the shoulders (and sometimes extend to below) of the tank. I guess that's about the right place for the shoulder and everything above it are valves and electronics. Vertical stripes can also be used.

Still not a fan of grape soda cans, but as long as it has markings on it I'll be happy.

Maybe add a way to allow hazard markings to be applied to any tank/can, in case somebody down the road decide to invent a gay pride gas.

chaoko99 commented:
As a secondary: I really can't think of many usage cases where this may end up being a problem. 

-An idiot fills a tank in atmosia (But engineering EVA is much closer, and there are two oxygen tanks on the way.
-An idiot attempts to fill their gas in the engine room ( W H Y )
- Someone goes into toxins storage to fill their tanks (This door has opened maybe twice. Ever.)
- Someone attempts to fill their tanks at the bluespace engine (Why are you even there?)
- Someone mistakenly fills the engine with oxygen, "because phoron looks like an oxygen tank now". See, that last one I question... If someone sees the two ext to each other, their mind might waver just a little, enough for them to check tooltips... I just don't see that being an issue. That phoron tank in the engine room is for emergency cooling... Muscle memory takes over. I'm not screwing with the map here.
Those are the only instances of phoron that come to mind...

chaoko99 commented:
Feel free to ignore most of that comment, but I will go ahead and make some hazard stripe sprites to overlay onto other cans.

Also it's supposed to just be red and yellow -- That black line is just the outline of the top of the can.

MistakeNot4892 commented:
Honestly I just think they look inappropriate for a dangerous fuel substance. Nothing about them says 'this shit is a hazard and will explode'.

Kasuobes commented:
Two pics for how trimix stripes are used and placed on tanks:
![](https://upload.wikimedia.org/wikipedia/commons/2/20/IMCA_Trimix_shoulder.svg)
![](https://upload.wikimedia.org/wikipedia/commons/e/eb/IMCA_Trimix_shoulder_quartered.svg)

Brown = helium
Black = nitrogen
White = oxygen

If you do use grape soda for phoron tanks, make sure you slap bright yellow hazard signs on it.

chaoko99 committed:
Adds hazard stripes to the grape soda can

Also adds a hazard stripe overlay for people who feel the need to use it on whatever gasses they add.

chaoko99 commented:
![image](https://cloud.githubusercontent.com/assets/4706052/24068242/fb319abc-0b46-11e7-80bd-9af5b03e457c.png)
I've settled on this specific design. A bit more obviously a hazard marker than two rings.

chaoko99 commented:
Did I say decided? No, I meant to say "I'm an indecisive shit"

![image](https://cloud.githubusercontent.com/assets/4706052/24068266/b544af2a-0b47-11e7-95e6-dee1ab444116.png)
@Kasuobes How's this?

Kasuobes commented:
![](http://www.backdrops.com.au/wp-content/uploads/2014/04/CQ020-Circus-Big-Top.jpg)

All you need now are spinning animations, clowns mimes and circus animals.

Make the stripes wider maybe, so you see 2-4 stripes per side.

chaoko99 committed:
Prettier hazard stripes

These ones actually look hazardous.

chaoko99 committed:
updated changelog.

chaoko99 commented:
It started off looking like ronald mc'donald's socks, so I suppose I couldn't quite escape the circus— Oh well.

Kasuobes commented:
Well clearly circus elements are toxic and flammable so I don't see the problem there.

chaoko99 committed:
Changelog

chaoko99 committed:
Makes the hazard overlays actually look hazardous.

chaoko99 committed:
Changelog.

Travis pls stop

chaoko99 committed:
Cleaned up hazard stripes, cleaned up hazard canister sprite

chaoko99 commented:
Why not clean up those hazard can sprites while we're down there?

![image](https://cloud.githubusercontent.com/assets/4706052/24068634/0aefe738-0b51-11e7-96a5-bbdfb83d45e5.png)

Final
![image](https://cloud.githubusercontent.com/assets/4706052/24068645/4035d60a-0b51-11e7-9799-5436e3626569.png)
I might just mess with that shear a little though.

mustafakalash commented:
But...why though?

chaoko99 commented:
Because I'm driven primarily by things that really bother me, instead of - say - Things like food, water, or other baseline needs such as success and self actualization. 

Nope. Just shit that bothers me. Primarily the fact that almost every other form of phoron is either purple or darkish pink... I want the gasses to reflect that.

BlueNexus commented:
So your sole reason for making these changes is that "you don't like it"? 

We already discussed this in the lore channel.

Just make everything phoron-related orange. I'll help you with the code side of things. It's just a few variables to make things orange, compared to a bunch of sprite changes for purple.

chaoko99 commented:
This was a dumb idea. Some good stuff came out of this but... Eh. Closing.
